arch/src/aarch64/fdt.rs

```diff
@@ -231,39 +231,72 @@ fn create_memory_node(
     guest_mem: &GuestMemoryMmap,
     numa_nodes: &NumaNodes,
 ) -> FdtWriterResult<()> {
+    // See https://github.com/torvalds/linux/blob/master/Documentation/devicetree/booting-without-of.txt#L960
```

Comment:
This is a broken link. Also I would suggest that you can use a link to this file instead of to a specific line (as the file might be modified which would lead to a wrong position).

Reply:
Yes, or you could link with a hash included.